pkg/cli/flags.go

```diff
@@ -266,6 +266,8 @@ func init() {
 		// TODO(peter): Decide if we want to make the lightstep flags visible.
 		if strings.HasPrefix(flag.Name, "lightstep_") {
 			flag.Hidden = true
+		} else if flag.Name == "no-redirect-stderr" {
+			flag.Hidden = true
```

Comment:
I've made `--no-redirect-stderr` flag hidden. I needed this flag to fix the `test_sql_mem_monitor.tcl` test. Should the flag be visible? Or should we recommend folks use `--logtostderr` if they do not want stderr redirected?